doc/cli/npm-doctor.md

```diff
@@ -0,0 +1,17 @@
+npm-doctor(1) -- Check your environments
+========================================================
+
+## SYNOPSIS
+
+    npm doctor
+
+## DESCRIPTION
+
+This command helps you when you get struck by environments which
+potentially have a problem. Some of the questions are same with
+we ask in the ISSUE_TEMPLATE.md.
```

Comment:
That's absolutly good idea, and I'd like to have the feature in `npm report` command which have been proposaling at https://github.com/npm/npm/issues/5214. I'm planning to use the `ISSUE_TEMPLATE.md` as a acutual template, but let's separete the reporting part from this diagnosis part to make the initial scope clear and simple.